src/components/operations/operation-details/ko/runtime/operation-console.ts

```diff
@@ -473,23 +524,14 @@ export class OperationConsole {
 
             this.responseStatusCode(response.statusCode.toString());
             this.responseStatusText(responseStatusText);
-            this.responseBody(response.toText());
-
-            const responseHeaders = response.headers.map(x => {
-                const consoleHeader = new ConsoleHeader();
-                consoleHeader.name(x.name);
-                consoleHeader.value(x.value);
-                return consoleHeader;
-            });
-
-            const contentTypeHeader = responseHeaders.find((header) => header.name().toLowerCase() === KnownHttpHeaders.ContentType.toLowerCase());
+            this.responseBody(response.body);
 
-            if (contentTypeHeader) {
-                if (contentTypeHeader.value().toLowerCase().indexOf("json") >= 0) {
+            if (response.contentTypeHeader) {
+                if (response.contentTypeHeader.toLowerCase().search("json") !== -1) {
```

Comment:
`response.contentTypeHeader.toLowerCase().search("json") !== -1` can be just `response.contentTypeHeader.toLowerCase().includes("json")`